native/cocos/bindings/event/EventDispatcher.cpp

```diff
@@ -331,6 +338,23 @@ void EventDispatcher::dispatchResizeEvent(int width, int height) {
     EventDispatcher::dispatchCustomEvent(EVENT_RESIZE, 0);
 }
 
+void EventDispatcher::dispatchResizeEvent(const WindowEvent &windowEvent) {
```

Comment:
Why need to overload dispatchResizeEvent?

Reply:
`dispatchResizeEvent` needs a windowId which is in `WindowEvent`, overloading it because of compatibility.